ajax/libs/hack-font/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "hack-font",
+  "filename": "web/hack.min.css",
+  "version": "3.000",
```

Comment:
Seem a little strange here. Normally it's like https://github.com/source-foundry/Hack/blob/master/package.json#L3. Maybe need to ask the author if he can improve this?